qa/tasks/cephfs/test_nfs.py

```diff
@@ -744,3 +744,27 @@ def test_non_existent_cluster(self):
         except CommandFailedError as e:
             if e.exitstatus != errno.ENOENT:
                 raise
+
+    def test_nfs_export_with_invalid_path(self):
+        """
+        Test that nfs exports can't be created with invalid path
+        """
+        self._test_create_cluster()
+
+        # test non-existent path
+        with self.assertRaises(CommandFailedError):
+            self._create_export(
+                export_id='123', create_fs=True,
+                extra_cmd=['--pseudo-path', self.pseudo_path, '--path',
+                           '/non_existent_dir'])
+
+        # test export creation at a filepath
+        self.ctx.cluster.run(args=['sudo', 'chmod', '1777', '/'])
+        self.ctx.cluster.run(args=['touch', '/testfile'])
```

Comment:
For the test, don't you need to create the file in ceph file system followed by creating an export with the path to the file?

Reply:
it was first created at `/mnt/` but since it doesnt work, i was experimenting with this testcase last night

Reply:
this has been fixed, `_test_create_cluster()` didn't create FS and thus could not locate path, it has been added to the testcase and works as expected.